wordpress-mobile/WordPress-iOS
# [Site Design Revamp] Re-enable large recommended theme screenshots

Fixes #18971

## Description

This PR re-enables the larger screenshots disabled as a work-around in https://github.com/wordpress-mobile/WordPress-iOS/pull/18879. This PR also fetches the screenshots for the theme picker using the dimensions from the "recommended" sizes (rather than "category" sizes). Other changes this PR relies on are on the API side (`D83271-code`).

### Testing steps

Pre-conditions:

* Device should be connected to a WPCOM sandbox
* The patch should be applied to the sandbox from `D83271-code`

1. Navigate to the site creation flow
2. Choose an Intent (e.g. "Food")
3. Expect the recommended for you screenshots to be larger
4. Expect the screenshots to be loaded with proper display resolution (i.e. not pixelated)
   * It can be helpful to inspect traffic (e.g. in Charles) to ensure that properly sized screenshots are requested.
   * Inspecting traffic can also help confirm that the returned screenshots are the correct size (limited to 1200 width).

#### Page picker

The page picker should not be affected by these changes (though backend changes in the API may result in changes to the fetched screenshot URLs)

## Regression Notes
1. Potential unintended areas of impact
Page picker

6. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual tests

7. What automated tests I added (or what prevented me from doing so)
Establishing the required testing apparatus for this would be out of scope for this task.

PR submission checklist:
